docker-compose.yml

```diff
@@ -286,6 +286,10 @@ services:
       test: rabbitmq-diagnostics -q alarms
       interval: 30s
       timeout: 3s
+    deploy:
+      resources:
+        limits:
+          memory: 512M
```

Comment:
Maybe I am just confused because I was not part of the conversation, but: Why is the max memory usage of the container times 1,5 the minimum required free disk space? Is that to be able to dump the memory to disk in case of a certain event?

Reply:
I think because the rabbitmq-prepare-for-production guideline tells to do so. 
Reasoning probably is: With X RAM RabbitMQ can process messages up to amount X. As rabbitMQ guarantees persistence, it therefore needs X+overhead disk space.

Reply:
Exactly as @ppuetsch said. This is the reason: https://www.rabbitmq.com/production-checklist.html#resource-limits-disk-space